src/default_environment.rs

```diff
@@ -218,9 +240,24 @@ pub fn default_env() -> Env {
             let start = require_int_parameter("range", args, 0)?;
             let end = require_int_parameter("range", args, 1)?;
 
-            Ok(Value::List(
-                (start..end).map(|i| Value::Int(i)).collect::<List>(),
-            ))
+            cfg_if! {
+                if #[cfg(feature = "bigint")] {
+                    let mut i = start.clone();
+                    let mut res = Vec::with_capacity((end.clone() - start)
+                        .to_usize()
+                        .ok_or(RuntimeError::new("Failed converting `BigInt` to `usize`"))?
+                    );
+
+                    while i <= end {
+                        res.push(i.clone());
+                        i += 1;
+                    }
+
+                    Ok(Value::List(res.into_iter().map(Value::Int).collect::<List>()))
+                } else {
+                    Ok(Value::List((start..=end).map(Value::Int).collect::<List>()))
```

Comment:
Oh wow, was this logic broken before? (`start..end` vs `start..=end`)

Reply:
Actually- I think the exclusive behavior was intended, since this is how ranges tend to work in other languages:
```
(range 0 10)
'(0, 1, 2, 3, 4, 5, 6, 7, 8, 9)
```

So let's remove the `=` here, and also change the `<=` to `<` in the `BigInt` version